frontend/src/components/MeetingRoom/EmojiGrid/EmojiGridMobile.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Grid, Grow, Portal } from '@mui/material';
+import { ClickAwayListener } from '@mui/base';
+import { ReactElement } from 'react';
+import SendEmojiButton from '../SendEmojiButton';
+import emojiMap from '../../../utils/emojis';
+
+export type EmojiGridMobileProps = {
+  handleClickAway: (event: MouseEvent | TouchEvent) => void;
+  isEmojiGridOpen: boolean;
+  isParentOpen: boolean;
+};
+
+/**
+ * EmojiGridMobile Component
+ *
+ * Displays a grid of emojis for devices with small viewports.
+ * @param {EmojiGridMobileProps} props - the props for the component
+ *  @property {(event: MouseEvent | TouchEvent) => void} handleClickAway - handles clicking away from the emoji grid
+ *  @property {boolean} isEmojiGridOpen - whether the component is open
+ *  @property {boolean} isParentOpen - whether the ToolbarOverflowMenu is open
```

Comment:
I think we might want to rename this, given the description. 
Is ToolbarOpen or something should make sense